projects/demo/src/modules/decorators/required-setter/import/example-decorator.txt

```diff
@@ -3,8 +3,8 @@
     quantity => Number.isInteger(quantity) && quantity >= 10,
     'Should be integer number more than min value',
 )
-set quantity(quantity: number) {
-    this.items = new Array(quantity).fill(Math.floor(Math.random() * Math.floor(100)));
+set quantity(quantity: number | undefined) {
+    this.items = new Array(quantity ?? 0).fill(Math.floor(Math.random() * Math.floor(100)));
 }
```

Comment:
`new Array(undefined) // = [undefined]`
This led to incorrect component behavior. When `quantity === undefined` there was one heart icon displayed in the example.

Reply:
Idea behind `tuiRequiredSetter` is to not let `undefined` through and notify developer that somehow they passed `undefined` as input and they should probably check their logic. looks like the flaw is someplace else.

Reply:
On a sidenote, instead of `| undefined` we write `?: number` as type.

Reply:
Ok then, let's investigate why `@tuiRequiredSetter` doesn't work properly